be/src/exec/pipeline/scan/olap_scan_operator.cpp

```diff
@@ -64,8 +65,16 @@ void OlapScanOperator::do_close(RuntimeState* state) {
     _ctx->unref(state);
 }
 
+OlapScanOperator::~OlapScanOperator() {
+    if (!_opened_tablet_ids.empty()) {
+        VLOG(1) << "close rowsets for " << _opened_tablet_ids << " query:" << print_id(_queryid);
+        _opened_tablet_ids.clear();
```

Comment:
Why clear it rather than release the memory directly?

Reply:
This pr only adds log, not changing code behavior,  `~OlapScanOperator` destructor should do the job, so I think `_opened_tablet_ids.clear();` is also unnecessary